### ISSUE TITLE
Add Dark Theme to Blog Page 

### ISSUE THREAD

**Issue by bkrayaguru931** [MonalikaPatnaik/UMatter#264]
The blog page lacks a dark theme, limiting user accessibility and customisation preferences.

I would like to work on this issue under GSSoC'23.

**Comment by Mayank1170** [comment]
Hey @bkrayaguru931 are you working on this or i may carry it forward ?

**Comment by bkrayaguru931** [comment]
> Hey @bkrayaguru931 are you working on this or i may carry it forward ?

I will push today

**Comment by MonalikaPatnaik** [comment]
Hey @bkrayaguru931 what's the status??

**Comment by bkrayaguru931** [comment]
> Hey @bkrayaguru931 what's the status??
> 

Hey I'm sorry for not fixing this issue till today. Actually I was getting some error and due to ongoing college exams I didn't get time to fix it. 

But I will surely fix and push it by tomorrow.

**Comment by MonalikaPatnaik** [comment]
Hey.. it's okay..Just let me know if u are unable to do it!
